posts/2021/05/may-status-update.md

```diff
@@ -0,0 +1,101 @@
+<!--
+.. title: HPy Status Update
+.. slug: may-status-update
+.. date: 2021-05-07 10:00:00 UTC
+.. author: hodgestar
+.. tags:
+.. category:
+.. link:
+.. description:
+.. type: text
+-->
+
+
+HPy's first git commit was on Friday, 12 July 2019. Despite the crazy state
+of the world, HPy has come a long way since then. This is a quick update
+to say where we are now and what's planned for the immediate future.
+
+<!--TEASER_END-->
+
+What can one do with HPy right now?
+===================================
+
+One can already port non-trivial Python extensions to the HPy API, but
```

Comment:
```suggestion
One can already port non-trivial Python extensions compiled with the HPy API, but
```

Reply:
I think one "ports to"? One can't "port compiled with".